src/test/java/core/basesyntax/dao/StorageDaoTest.java

```diff
@@ -0,0 +1,82 @@
+package core.basesyntax.dao;
+
+import static core.basesyntax.db.Storage.storage;
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.dao.impl.StorageDaoImpl;
+import core.basesyntax.model.Fruit;
+import java.util.NoSuchElementException;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class StorageDaoTest {
+    private static StorageDao storageDao;
+    private Fruit defaultFruit;
+
+    @BeforeClass
+    public static void beforeClass() {
+        storageDao = new StorageDaoImpl();
+    }
+
+    @Before
+    public void setUp() {
+        defaultFruit = new Fruit("banana", 100);
+    }
+
+    @Test
+    public void storageDao_add_Ok() {
+        storageDao.add("banana", 100);
+        Fruit expected = defaultFruit;
+        Fruit actual = storage.get(0);
+        //new fruit
+        assertEquals(expected.getName(), actual.getName());
+        assertEquals(expected.getAmount(), actual.getAmount());
+        //existing fruit
```

Comment:
Uncle Bob says: don't explain with comments what code explains better)
Do we really need those comments?

Reply:
Forgot to remove them, will do